src/components/Editor/CallSites.js

```diff
@@ -144,15 +145,34 @@ class CallSites extends Component {
   }
 
   render() {
-    const { editor, callSites, selectedSource } = this.props;
+    const { editor, callSites, selectedSource, symbols } = this.props;
     const { showCallSites } = this.state;
+
     let sites;
     if (!callSites) {
       return null;
     }
 
+    const { line } = editor.codeMirror.getCursor();
```

Comment:
I think `getCursor` gives us the position of the blinking cursor, and not mouse position, I am not sure though.

Reply:
It does give the position of the blinking cursor, which was my intention for this first iteration. Do you happen to know of any way to get the mouse hover position?

Reply:
I did read the docs but couldn't find the mouse position, I think we need to come up with our own way to capture mouse position.

Reply:
Let's table the mouse position experiment for now.  Let's just focus on clicking the icon adding the column breakpoint, and then having different states like enabled, disabled, off, as well as dark mode 💯